pkg/api/v20231122/openshiftcluster.go

```diff
@@ -68,7 +68,7 @@ type OpenShiftClusterProperties struct {
 	WorkerProfilesStatus []WorkerProfile `json:"workerProfilesStatus,omitempty" swagger:"readOnly"`
 
 	// The cluster API server profile.
-	APIServerProfile APIServerProfile `json:"apiserverProfile,omitempty"`
+	APIServerProfile APIServerProfile `json:"apiserverProfile,omitempty" swagger:"readOnly"`
```

Comment:
Same as above, instead of `Profile`, let's do the specific field of the URL (L252)